ironic/drivers/modules/drac/management.py

```diff
@@ -387,5 +400,81 @@ def known_good_state(self, task):
         node = task.node
 
         client = drac_common.get_drac_client(node)
+        if client.is_lifecycle_in_recovery():
+            get_lcc_out_of_recovery_mode(node)
```

Comment:
It will be good to add log.info("Life cycle controller is in recovery mode")

Reply:
Sure. Will add this log message.